onprc_ehr/resources/web/onprc_ehr/panel/AssignmentFilterType.js

```diff
@@ -65,7 +65,7 @@ Ext4.define('ONPRC_EHR.panel.AssignmentFilterType', {
                 store: {
                     type: 'labkey-store',
                     schemaName: 'onprc_ehr',
-                    sql: 'SELECT distinct lastname FROM onprc_ehr.investigators WHERE lastname is not null',
+                    sql: 'SELECT distinct lastname FROM onprc_ehr.investigators WHERE enddate is null',
```

Comment:
This change seems unrelated. Is it something you want to include here?

Reply:
NO let me remove it from the build

Reply:
File restored to original state and submitted

Reply:
@jonesgaohsu you added a comment but didn't actually back out the change to the SQL.